packages/core/package.json

```diff
@@ -68,6 +70,7 @@
   "dependencies": {
     "deepmerge": "^4.0.0",
     "hoist-non-react-statics": "^3.3.1",
+    "pcln-types": "^0.0.1",
```

Comment:
I am assuming `pcln-types` is a project that will pulled in optionally separately by each project to get type declarations. So why do we need it as a dependency here? Should it not be a `devDependency` instead?

Reply:
☝️

Reply:
Why should each project separately install pcln-types, if this is a dependency here, then installing pcln-types is not required in other projects. @huzefakagdi @craigpalermo  Correct me if I understood this differently.

Reply:
@SrikanthDarma types are always imported conditionally for all packages that do not export types directly on build time. Also, if you see the `prepublish` script task above, we are copying the types declarations directly in the `pcln-design-system` package. Which seems one more reason IMHO that this package may not be required as a `dependency`.

Reply:
i think you want the option to install them because you may not have a typescript project and you may not care about getting the types. also, it's nice to a have a central place to view all your types imo.

Reply:
That is my what I am thinking as well Jeff

Reply:
Yeah, I'm not sure if `pcln-types` should be a dependency or devDep of `pcln-design-system`. My thinking here was that the types would be installed in `node_modules`, then on `prepublish` we'd copy them into `dist/types` and reference them via the `"types"` property in `package.json`.

 I _think_ it'd just need to be a devDep because they're only needed when publishing; consumers of `pcln-design-system` would get the `types/` dir along with `cjs`, `esm`, and `es` in `dist/` and the `"types"` property in `package.json` would point to that.

Reply:
+1 for `devDep`